Sources/Build/ToolProtocol.swift

```diff
@@ -142,6 +142,6 @@ struct SwiftCompilerTool: ToolProtocol {
         stream <<< "    enable-whole-module-optimization: "
             <<< Format.asJSON(target.buildParameters.configuration == .release) <<< "\n"
         stream <<< "    num-threads: "
-            <<< Format.asJSON("\(SwiftCompilerTool.numThreads)") <<< "\n"
```

Comment:
We should remove the `numThreads` variable since I don't think it's used anywhere else.

Reply:
Sorry, where do you want to remove it from?